### PR TITLE
firestarter: init at 1.7.3

### PR DESCRIPTION
**Pull request by astro**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @Mic92

### CODE REVIEW

File: pkgs/applications/misc/firestarter/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, python3, cudatoolkit,
+  withCuda ? true
+}:
+
+with stdenv.lib;
```
Comment (mmahut) [inline pkgs/applications/misc/firestarter/default.nix]: Can you please try to use ```buildPythonPackage```?

File: pkgs/applications/misc/firestarter/default.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+stdenv.mkDerivation rec {
+  name = "firestarter-${version}";
```
Comment (mmahut) [inline pkgs/applications/misc/firestarter/default.nix]: Please consider using ``${pname}``.

File: pkgs/applications/misc/firestarter/default.nix (hunk 1 above)
Comment (astro) [reply pkgs/applications/misc/firestarter/default.nix]: It's not a python package?

File: pkgs/applications/misc/firestarter/default.nix (hunk 2 above)
Comment (astro) [reply pkgs/applications/misc/firestarter/default.nix]: de9f0e03ec9

File: pkgs/applications/misc/firestarter/default.nix (hunk 1 above)
Comment (mmahut) [reply pkgs/applications/misc/firestarter/default.nix]: Sorry, commented on wrong PR :)

**Comment by mmahut** [comment]
Thank you, just please squash your commits into one before merging this.

**Comment by mmahut** [comment]
@GrahamcOfBorg build firestarter

**Comment by astro** [comment]
> Thank you, just please squash your commits into one before merging this.

Rebased and squashed.

**Comment by mmahut** [comment]
@GrahamcOfBorg build firestarter

**Pull request merged**